### ISSUE TITLE
Missing property "aria-description"

### ISSUE THREAD

**Issue by vekunz** [sveltejs/language-tools#1975]
### Describe the bug

If I add the `aria-description` property to an HTML Tag, the svelte-check command fails with an error, that this property does not exists, although it does exist.

```
Error: Argument of type '{ class: string; "aria-description": string; }' is not assignable to parameter of type 'HTMLProps<"p", HTMLAttributes<any>>'.
  Object literal may only specify known properties, and '"aria-description"' does not exist in type 'HTMLProps<"p", HTMLAttributes<any>>'. (ts)
```

### Reproduction

```svelte
<p aria-description="Something"></p>
```

### Expected behaviour

`svelte-check` should accept this property on any HTML element.

### System Info

- OS: Windows
- IDE: WebStorm

### Which package is the issue about?

svelte-check

### Additional Information, eg. Screenshots

Documentation: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description

Specification: https://w3c.github.io/aria/#aria-description

**Comment by dummdidumm** [comment]
`aria-description` is still in draft mode and therefore not standardized/implemented by all assistive technologies yet, which is why the typing is not present yet. If you still want to use it, you can enhance the typings [as described here](https://github.com/sveltejs/language-tools/blob/master/docs/preprocessors/typescript.md#im-using-an-attributeevent-on-a-dom-element-and-it-throws-a-type-error)
